apps/src/templates/sectionsRefresh/SingleSectionSetUp.jsx

```diff
@@ -25,14 +30,16 @@ export default function SingleSectionSetUp({
           onChange={e => updateSection('name', e.target.value)}
         />
       </label>
-      <MultiSelectGroup
-        label={i18n.chooseGrades()}
-        name="grades"
-        required={true}
-        options={gradeOptions}
-        values={section.grade || []}
-        setValues={g => updateSection('grade', g)}
-      />
+      {participantType !== 'teacher' && (
```

Comment:
Can we use the constant to check for this teacher value? Import would be something like:
import {ParticipantAudience} from '@cdo/apps/generated/curriculum/sharedCourseConstants';

Also have a small nit, I think I'd slightly prefer the non-negated version like:
'participantType === ParticipantAudience.student &&'

Reply:
I actually think that 'nit' is really good because we have a facilitator audience too that shouldn't get the grade selector which I missed initially.